src/views/ApartmentApp/components/StudentApplication/index.js

```diff
@@ -47,71 +55,84 @@ const StudentApplication = ({ userProfile, authentication }) => {
   const [submitDialogOpen, setSubmitDialogOpen] = useState(false); // Use this for submitting app (later feature)
   const [snackbarOpen, setSnackbarOpen] = useState(false);
   const [snackbarText, setSnackbarText] = useState('');
-  const [snackbarSeverity, setSnackbarSeverity] = useState('');
+  const [snackbarSeverity, setSnackbarSeverity] = useState('info');
   const [saveButtonAlertTimeout, setSaveButtonAlertTimeout] = useState(null);
 
   /**
    * Loads the user's saved apartment application, if one exists
    */
-  const loadSavedApplication = useCallback(async () => {
-    // TODO: Implement this once save/load of application data has been implemented in the backend
-    setLoading(true);
-    // Check if the current user is on an application. Returns the application ID number if found
-    let newApplicationID = null;
-    try {
-      newApplicationID = await housing.getCurrentApplicationID();
-      //! DEBUG
-      console.log('Retrieved Application ID: ' + newApplicationID);
-      if (newApplicationID === null || newApplicationID === -1) {
-        // Intentionally trigger the 'catch'
-        throw new Error("Invalid value of 'newApplicationID' = " + newApplicationID);
-      }
-      setApplicationID(newApplicationID);
-      let applicationDetails = await housing.getApartmentApplication(newApplicationID);
-      if (applicationDetails) {
-        setDateSubmitted(applicationDetails.DateSubmitted ?? null);
-        setDateModified(applicationDetails.DateModified ?? null);
-        setEditorUsername(applicationDetails.EditorUsername ?? null);
-        if (applicationDetails.Applicants) {
-          applicationDetails.Applicants.forEach(async (applicantInfo) => {
-            const newApplicantProfile = await user.getProfileInfo(applicantInfo.Username);
-            setApplicants((prevApplicants) =>
-              prevApplicants.concat({
-                Profile: newApplicantProfile,
-                OffCampusProgram: applicantInfo.OffCampusProgram,
-              }),
-            );
-          });
-        }
-      }
-    } catch {
+  useEffect(() => {
```

Comment:
I'm curious: what does this do differently as a useEffect that is better than the useCallback method?

Reply:
The function needed to be setup as a `useCallback` so that the `useEffect` below it could call that function, but also have that function be callable from elsewhere within the functional component. We have since figured out that we won't need to call that function elsewhere anymore, so I basically just changed it back to how I had originally coded it and added the `isSubscribed` stuff to prevent the errors we talked about earlier.

To try to say that less confusingly,
The `useCallback` was basically an adaptation to allow the loadApp function to be called from both the `useEffect` and from other places as well, 
but since we no longer needed that I decided to simplify it again since I needed to refactor the internal structure of it all anyways for the other bug fix

Reply:
See lines 105-107 of the deleted side of the file comparison

Reply:
That first part was clear enough, actually, but thank you for simplifying. That makes sense then.